README.md

```diff
@@ -32,7 +32,7 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
   go build -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
   ```
 
-- Create the project using gin-templates. Note that the **main** branch is used.
+- Create the project using gin-templates. Note that the **main** branch is used. Refer to [this wiki page](https://github.com/nimblehq/gin-templates/wiki/Commands) for instructions on how to use a different branch.
```

Comment:
Can we change the text like this?
```suggestion
- Create the project using gin-templates. Note that the **main** branch is being used by default. Refer to [this wiki page](https://github.com/nimblehq/gin-templates/wiki/Commands) for instructions on how to use a different branch.
```

Reply:
Good catch. Fixed in 65a9b78c2557ed0af301851210e84459452c4c1b